### PR TITLE
fix: remove envsetup from files. new paths. record pilot3 

### PR DESCRIPTION
**Pull request by bms63**
* All new paths to read in
* removed all envsetup files
* removed pilot1 and envsetup from lock file
* recorded pilot3 in lock file
* various tidying up

### CODE REVIEW

File: .Rprofile [hunk 1]
```diff
@@ -1,6 +1,5 @@
 source("renv/activate.R")
 if (Sys.getenv("GITHUB_ACTIONS") == "") {
-  envsetup::rprofile(config::get(file = "_envsetup.yml", config = "prod"))
```
Comment (laxamanaj) [inline .Rprofile:3]: Thanks for removing this @bms63 .  The adam and tlf scripts still work without envsetup.

File: submission/programs/adadas.r [hunk 2]
```diff
@@ -27,17 +25,20 @@ library(stringr)
 library(xportr)
 library(pilot3)
 
-dm <- haven::read_xpt(file.path(path$sdtm, "dm.xpt"))
-qs <- haven::read_xpt(file.path(path$sdtm, "qs.xpt"))
-adsl <- haven::read_xpt(file.path(path$adam, "adsl.xpt"))
+sdtm_path <- "./submission/sdtm/"
+adam_path <- "./submission/adam/"
```
Comment (laxamanaj) [inline submission/programs/adadas.r:29]: I just want to ensure that these paths won't stay hard coded in each of the adam programs.   

if needed you can run this first : 
```
path <- list(
  sdtm = "./submission/sdtm",   # Modify path to the sdtm location
  adam = "./submission/adam",    # Modify path to the adam location
  output = "./submission/output"    # Modify path to the output location
)


```

then you should be able to run the prior version using `path$sdtm` or `path$adam`.

**Review by laxamanaj — CHANGES_REQUESTED** [review]
Added some comments.  Thanks, @bms63 !

File: submission/programs/adadas.r (hunk 2 above)
Comment (bms63) [reply submission/programs/adadas.r:29]: sure we can do that, but it still hardcoded - just in a list.  I find accessing a list with `$` to be slightly confusing for new folks.    Happy to change it if that is what you want to do

Comment (laxamanaj) [reply submission/programs/adadas.r:29]: Thanks, @bms63 .  Changing it back will be consistent with Pilot 1 as we updated in #116 and #117 .

**Review by DeclanHodges — APPROVED** [review]
Looks good to me, thanks @bms63.

**Comment by bms63** [comment]
@laxamanaj i made the requested updates. what do you think about removing the modification history.  Seems pointless with using github/git

image

**Comment by laxamanaj** [comment]
> @laxamanaj i made the requested updates. what do you think about removing the modification history. Seems pointless with using github/git
> 
> image

Thanks, @bms63 .  I agree, since git captures all changes and who updates it anyway.  For this, I think it's fine to leave the modification history.  Shows that our good programming practice in the industry is still in tact. We still do it for our 'real' studies anyway, but you have a point that eventually, git will capture all of this info automatically.

File: vignettes/project_setup.Rmd
Comment (laxamanaj) [inline vignettes/project_setup.Rmd:1]: Looks good.

File: renv.lock
Comment (laxamanaj) [inline renv.lock:1]: thanks for updating this.  I still don't understand what I was not doing right to get pilot3 package in the lock file. I'll catch up with you for more details on these steps.

File: inst/WORDLIST
Comment (laxamanaj) [inline inst/WORDLIST:1]: I could use more learning on the changes made here as well.  Thanks for updating, @bms63 !

File: DESCRIPTION
Comment (laxamanaj) [inline DESCRIPTION:1]: I could use more learning on the changes made here as well.  Thanks for updating, @bms63 !

File: .Rprofile
Comment (laxamanaj) [inline .Rprofile:1]: Looks good. Thanks for removing.

File: .Rbuildignore
Comment (laxamanaj) [inline .Rbuildignore:1]: Looks good.  thanks for updating.

**Review by laxamanaj — COMMENTED** [review]
I'll continue reviewing the rest of the programs tomorrow with the path updates.  Thanks again for the updates here, @bms63 .

File: submission/programs/path.r
Comment (bms63) [inline submission/programs/path.r:1]: @laxamanaj  I had same thought to add something like this.  However, you might need to put `source("path.r")` at the top of each file so each file can run as standalone if needed.

**Review by laxamanaj — APPROVED** [review]
updates are good to go.

File: submission/programs/path.r
Comment (laxamanaj) [reply submission/programs/path.r:1]: Unfortunately, in our project repo, my batch_ruh.sh won't run anymore, so we have to run every ADaM script one-by-one, interactively, which is the instruction in step 5 of the ADRG anyway, similar to Pilot 1.

**Pull request merged**